cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -20,47 +20,19 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/spf13/cobra"
-
 	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger/aries-framework-go/pkg/restapi/helper"
```

Comment:
helper is not a descriptive package name.

Reply:
This file is gone now, see my other comment in the thread regarding internal package